specifyweb/frontend/js_src/lib/components/SchemaConfig/Tables.tsx

```diff
@@ -127,13 +139,34 @@ export function TableList({
           );
         })}
       </Ul>
-      <Label.Inline>
-        <Input.Checkbox
-          checked={showHiddenTables}
-          onValueChange={setShowHiddenTables}
-        />
-        {wbPlanText.showAllTables()}
-      </Label.Inline>
-    </>
+    );
+  };
+
+  return (
+    <div className="flex flex-col items-start gap-2">
+      {tableList(presentFilteredTables)}
+      <Button.Save onClick={handleOpen}>{commonText.add()}</Button.Save>
+      {isOpen && (
```

Comment:
sorry, looks like you made it more complicated than what I meant
There should be a checkbox just like there was before. if you click the checkbox, the list above the checkbox excludes or includes the tables that have 0 forms - that's it - no need for dialogs

if you need an example of how such checkbox works, look at the base table selection dialog in the workbench mapper

Reply:
I followed Grant's requests :) 
I will create another version with the check box request and post 2 videos